server/src/bootstrap/index.ts

```diff
@@ -194,9 +198,100 @@ if (baseConfig.nodeEnv === Environment.Prod) {
+    $('meta[property="og:title"]').attr(
+      'content',
+      `${req.params.shortname.toUpperCase()} FAQ - AskGov`,
+    )
+    $('meta[property="og:url"]').attr(
+      'content',
+      `${req.protocol}://${req.hostname}${req.originalUrl.toLowerCase()}`,
+    )
+
+    const agencyPromise = async () => {
+      return await agencyService.findOneByName({
+        shortname: req.params.shortname,
+      })
+    }
+    ;(async () => {
+      const agency = await agencyPromise()
+      if (agency.isOk()) {
+        $('meta[name="description"]').attr(
+          'content',
+          `Answers from ${
+            agency.value.longname
+          } (${req.params.shortname.toUpperCase()})`,
+        )
+        $('meta[property="og:description"]').attr(
+          'content',
+          `Answers from ${
+            agency.value.longname
+          } (${req.params.shortname.toUpperCase()})`,
+        )
+      } else {
+        $('meta[name="description"]').attr(
+          'content',
+          `Answers from ${req.params.shortname.toUpperCase()}`,
+        )
+        $('meta[property="og:description"]').attr(
+          'content',
+          `Answers from ${req.params.shortname.toUpperCase()}`,
+        )
+      }
+      res.header('content-type', 'text/html').send($.html())
+    })()
+  })
+
+  app.get('/questions/:id', (req, res) => {
+    const $ = cheerio.load(index)
+
+    $('meta[property="og:type"]').attr('content', 'article')
+    $('meta[property="og:url"]').attr(
+      'content',
+      `${req.protocol}://${req.hostname}${req.originalUrl.toLowerCase()}`,
+    )
+
+    const postId: number = +req.params.id
+    const postPromise = async () => {
+      return await postService.getSinglePost(postId)
+    }
+    const answersPromise = async () => {
+      return await answersService.listAnswers(postId)
+    }
+    ;(async () => {
+      const post = await postPromise()
+      $('title').text(`${post.title} - AskGov`)
+      $('meta[property="og:title"]').attr('content', `${post.title} - AskGov`)
+      const answers = await answersPromise()
+      if (answers && answers.length > 0) {
+        const answerBody = sanitizeHtml(answers[0].body, {
+          allowedTags: [],
+          allowedAttributes: {},
+        })
+        $('meta[name="description"]').attr('content', `${answerBody}`)
+        $('meta[property="og:description"]').attr('content', `${answerBody}`)
+      } else {
+        $('meta[name="description"]').attr('content', `${post.description}`)
+        $('meta[property="og:description"]').attr(
+          'content',
+          `${post.description}`,
+        )
+      }
+      res.header('content-type', 'text/html').send($.html())
+    })()
+  })
+
+  app.get('*', (req, res) => {
+    const $ = cheerio.load(index)
+    $('meta[property="og:url"]').attr(
+      'content',
+      `${req.protocol}://${req.hostname}${req.originalUrl.toLowerCase()}`,
+    )
+    res.header('content-type', 'text/html').send($.html())
+  })
```

Comment:
I think this is slightly spurious, so maybe we can drop this =)

Reply:
okays